_posts/en/newsletters/2020-07-22-newsletter.md

```diff
@@ -0,0 +1,195 @@
+      mailing list an announcement of a new `##taproot-activation` IRC
+      room on the Freenode.net network.  The channel is [logged][irc
+      log] and saw an impressive amount of conversation from dozens of
+      participants during its first week.
+
+    - **Mailing list thread:** Anthony Towns posted a summary of
+      the recently-updated [BIP8][] activation method (see [Newsletter
+      #104][news104 bip8]) and of Towns's own new [bip-decthresh][]
+      activation proposal (based on Matt Corallo's "modern soft fork
+      activation" [post][msfa] in January, see [Newsletter #80][news80
+      activation]).  An interesting feature of the decreasing threshold
+      idea is that the amount of network hashrate needed to signal
+      readiness to enforce the new soft fork rules would decrease during
+      the final activation period, allowing miner activation with
+      perhaps as little as 60% miner support rather than the 95% miner
+      support needed during the first activation period.
+
+- **Bech32 address updates:** Russell O'Connor [posted][oconnor post] to
+  the Bitcoin-Dev mailing list a reply to Pieter Wuille's November post
+  (see [Newsletter #77][news77 bech32]) about amending BIP173 to only
+  allow witness programs of either 20 or 32 bytes, minimizing the risk
+  that typos would cause a user to irrecoverably send bitcoins to a
+  wrong address.  O'Connor made the counter-proposal that we could allow
+  any witness program that had five more characters in its address than
+  the next shortest address, up to the consensus-enforced length limit
+  on witness programs.  The counter-proposal didn't receive any
+  discussion on the mailing list, but a [PR][BIPs #945] to update BIP173
+  did receive several comments.  As of this writing, no conclusion has
+  been reached about what length restrictions to use.
+
+## Changes to services and client software
+
+*In this monthly feature, we highlight interesting updates to Bitcoin
+wallets and services.*
+
+- **Electrum adds Lightning Network and PSBT support:**
+  [Electrum's 4.0.1][electrum release notes] LN support includes local and
+  remote [watchtower][topic watchtowers] capabilities
+  as well as submarine swaps using Electrum Technologies' central server.
+  Electrum has also replaced their partial transaction format with PSBT.
+
+- **Lily Wallet initial release:**
+  [Lily Wallet][lily wallet site] has announced the initial v0.0.1-beta release
+  of their multi-account desktop multisig vault application. Lily Wallet creates
+  2-of-3 bech32 addresses and uses [HWI][topic hwi], HD wallets, and PSBT to transact.
+
+- **Zap 0.7.0 Beta released:** [Zap 0.7.0 Beta][zap 0.7.0] adds [spontaneous payments][topic
+  spontaneous payments], [payment probes][news30 probing], [multipath
+  payments][topic multipath payments], and [hold invoices][topic hold invoices]
+
+- **BTCPay Server 1.0.5.0 implements various standards:**
```

Comment:
Link to announcement: https://blog.btcpayserver.org/btcpay-server-1-0-5-0/ or github release page: https://github.com/btcpayserver/btcpayserver/releases/tag/v1.0.5.0 ?

Reply:
Added, thanks!